addons/website_sale_slides/i18n/website_sale_slides.pot

```diff
@@ -244,11 +249,6 @@ msgstr ""
 msgid "Website"
 msgstr ""
 
-#. module: website_sale_slides
-#: model_terms:ir.ui.view,arch_db:website_sale_slides.course_purchased_confirmation_message
-msgid "You have gained access to the following course(s):"
-msgstr ""
-
```

Comment:
Almost :smile: 

Actually when modifying strings in stable we don't remove the old term, to keep compatibility in case the client database is not properly updated.